app/protocol/constants.go

```diff
@@ -51,4 +54,6 @@ var (
 	KeyGuardian = sdk.NewKVStoreKey(GuardianStore)
 	KeyUpgrade  = sdk.NewKVStoreKey(UpgradeStore)
 	KeyAsset    = sdk.NewKVStoreKey(AssetStore)
+
+	BurnedCoinsAccAddr = sdk.AccAddress(crypto.AddressHash([]byte("burnedCoins")))
```

Comment:
Avoid defining the same constant twice.  We should move all the special address constants from `bank/keeper.go` to `auth/keeper.go`.